README.md

```diff
@@ -5,11 +5,14 @@ Plugin for Solr that allows users to filter locations using the Traveltime API.
 This is a standard Solr plugin.
 The plugin jar must be copied into the [solr lib directory](https://solr.apache.org/guide/8_4/libs.html#lib-directories)
 
-To use the plugin you **must** add a `queryParser` with the class `com.traveltime.plugin.solr.TraveltimeQParserPlugin`.
+To use the plugin you **must** add a `queryParser` with the class `com.traveltime.plugin.solr.TraveltimeQParserPlugin` or
+`com.traveltime.plugin.solr.TimeFilterQParserPlugin` (currently only available for solar version 8).
```

Comment:
Is it available with all solr 8 versions?

Reply:
Yes, we don't get more granular with solr